lib/typelixir/functions_extractor.ex

```diff
@@ -34,7 +34,7 @@ defmodule Typelixir.FunctionsExtractor do
   # ---------------------------------------------------------------------------------------------------
 
   defp extract({:@, [line: line], [{:spec, _, [{:::, _, [{fn_name, _, type_of_args}, type_of_return]}]}]} = elem, env) do
-    type_of_args = Enum.map(type_of_args, fn type -> PatternBuilder.type(type, %{}) end)
+    type_of_args = Enum.map(type_of_args || [], fn type -> PatternBuilder.type(type, %{}) end)
```

Comment:
What do you think about adding a test for this? As you did with the functions definitions

Reply:
@mcass19 Sure, I just added it, please check it now!